cypress/integration/group1/dashboard.ts

```diff
@@ -43,4 +44,14 @@ describe('Dockstore dashboard', () => {
       .should('have.attr', 'href')
       .and('include', 'getting-started-with-services');
   });
+  it('Registering new tool through Github redirects correctly', () => {
```

Comment:
Might be worth having a similar test for workflows and services too

Reply:
Added one for workflows. The button to register services seems to link straight to the github link so I could not add a test for it as cypress does not support different domain redirects